core/scss/components/lockup/_lockup--option-c.scss

```diff
@@ -0,0 +1,24 @@
+@charset 'UTF-8';
+
+.su-lockup--option-c {
+
+  .su-lockup__line3,
+  .su-lockup__line4 {
+    display: none;
+  }
+
+  .su-lockup__line1 {
+    font-size: 1.9rem;
+    padding-bottom: 0.2em;
```

Comment:
I remember we all discussed this, so apologies if this is inline with what we decided. but did we want to use the padding mixin for consistency even when only specifying an individual side for padding? If so, `@padding();` should be used here.

Reply:
🤦‍♂️ my bad. I forgot. Thanks for catching.